sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -235,6 +237,55 @@ public ServiceBusClientBuilder credential(String fullyQualifiedNamespace, TokenC
         return this;
     }
 
+    /**
+     * Sets the credential for the Service Bus resource.
+     *
+     * @param fullyQualifiedNamespace for the Service Bus.
+     * @param credential {@link TokenCredential} to be used for authentication.
```

Comment:
This overload does not use TokenCredential, so this comment should be updated. Same for the other overload that takes the AzureSasCredential

Reply:
Sorry, I missed the correction of comment. According to your comment in the new revision amendments have been made.